UnityProject/AddressablePackingProjects/SoundAndMusic/Assets/AddressableAssetsData/AssetGroups/Music.asset

```diff
@@ -239,6 +239,18 @@ MonoBehaviour:
     m_SerializedLabels: []
     m_MainAsset: {fileID: 0}
     m_TargetAsset: {fileID: 0}
+  - m_GUID: fab582363ff3d7c478a69bc0da678df2
+    m_Address: Assets/Prefabs/AI/captain_announcement.prefab
```

Comment:
I don't think these are music sounds, are they?

Reply:
huh that's weird